src/test/java/com/example/tournamentMaker/tournament/result/ResultServiceTest.java

```diff
@@ -68,18 +62,29 @@ void shouldUpdateSpecificStatisticInTeam() {
         assertEquals(23, specificStatistic.get(player2.getId()));
     }
 
-
-    private Team createTeam(String name) {
-        return new Team(name, createTournament());
-    }
-
-    private Tournament createTournament() {
-        return new Tournament("Tournament A", TournamentType.LEAGUE, Sport.FOOTBALL);
+    @Test
+    void shouldUpdateSpecificStatisticInTeamWhenNoPlayersInSpecificStatisticMap() {
+        //given
+        List<Integer> jerseyNumbersList = List.of(5, 5, 5, 5);
+        Team team = util.createTeam("Team A");
+        FootballPlayer player = util.createFootballPlayer(1L, "Jack", team, 5);
+        Map<Long, Integer> specificStatistic = new HashMap<>();
+        //when
+        when(playerRepository.findByJerseyNumberAndTeam(5, team)).thenReturn(Optional.of(player));
+        resultService.updateSpecificStatisticInTeam(jerseyNumbersList, team, specificStatistic);
+        //then
+        assertEquals(4, specificStatistic.get(player.getId()));
     }
 
-    private FootballPlayer createFootballPlayer(long id, String firstName, Team team, int jerseyNumber) {
-        FootballPlayer player = new FootballPlayer(firstName, "LastName", team, jerseyNumber, FootballPosition.MIDFIELDER);
-        player.setId(id);
-        return player;
+    @Test
+    void shouldContainExceptionWhenPlayerWithJerseyNumberNotFound() {
+        //given
+        List<Integer> jerseyNumbersList = List.of(5);
+        Team team = util.createTeam("Team A");
+        Map<Long, Integer> specificStatistic = new HashMap<>();
```

Comment:
skoro ta mapa jest pusta a musisz ja miec zeby przekazac do metody to nie ma sensu robic z niej zmiennej